### ISSUE TITLE
More clearly indicate the requirements for calculating indicators and indices

### ISSUE THREAD

**Issue by Zeitsperre** [Ouranosinc/xclim#1438]
### Problem

The primary method for determining what variables are needed to calculate a given index or indicator is to look at the call signature (sometimes *very* busy) and/or the `Parameters` entries. This can be annoying for users as they might want to calculate something and gloss over a variable that might be required.

### Potential Solution

Each index or indicator could list before the `Parameters` section a block that lists these requirements. e.g.:

```
Requires
--------
# <standard_name> (variable)
temperature (tas), precipitation_flux (pr), geopotential (z)
```

**Comment by Zeitsperre** [comment]
@SarahG-579462

Feel free to edit this issue and/or elaborate on how you'd like to see things change.

**Comment by huard** [comment]
I think we should stick with the numpy doc convention. https://numpydoc.readthedocs.io/en/latest/format.html

Question: does this relate to the HTML docs or the help you get within a Python shell ? If it's the HTML, we could possibly tweak the CSS to highlight some parameters.

**Comment by Zeitsperre** [comment]
Fixed with #1454
